project-service/api.md

```diff
@@ -0,0 +1,594 @@
+# Project Service External API
+
+## Users
+
+### Fetch Users
+
+Returns json data of all users as an array
+
+- URL
+  + `/users`
```

Comment:
I guess I was thinking it might come into use if the user needs a list of users to choose from to invite to a project or organization or something? Because right now we don't have a consistent way of tracking users. You could use email (I would have to save the email property from Auth0, which is no problem). If you will never need this, then of course it should be deleted.